.github/container/Dockerfile.jax

```diff
@@ -36,6 +36,10 @@ RUN --mount=type=ssh \
 EOF
 
 ADD build-jax.sh local_cuda_arch test-jax.sh /usr/local/bin/
+# Install bazelisk
+RUN ARCH="$(dpkg --print-architecture)" && \
+    wget -O /usr/bin/bazel https://github.com/bazelbuild/bazelisk/releases/latest/download/bazelisk-linux-${ARCH} && \
+    chmod +x /usr/bin/bazel
```

Comment:
I think `/usr/local/bin/bazel` would be more consistent with the rest of the container.